portfolio/src/main/webapp/script.js

```diff
@@ -51,7 +51,12 @@ function loadComments() {
         fetch('/data').then(response => response.json()).then((comments) => {
             const commentListElement = document.getElementById('comment-list');
             for(i = 0; i < numToLoad; i++){
-                commentListElement.appendChild(createCommentElement(comments[i]))
+                comment = createCommentElement(comments[i]);
+                if(comment){
+                    commentListElement.appendChild(comment);
+                }else{
+                    numToLoad++;
+                }
```

Comment:
Actually what happens is, if I ask for too many comments, I get:

script.js:76 Uncaught (in promise) TypeError: Cannot read property 'userName' of undefined
    at createCommentElement (script.js:76)
    at script.js:54

Because `comments[i]` returns `undefined` if `i` is greater than `comments.length`

Reply:
Alright that should be fixed